util/test/testUtil.h

```diff
@@ -44,7 +45,7 @@ void valuecheck(const Eigen::DenseBase<DerivedA>& a, const Eigen::DenseBase<Deri
 
 void valuecheck(double a, double b, double tolerance = 1e-8)
 {
-  if (abs(a - b) > tolerance) {
+  if (std::abs(a - b) > tolerance) {
```

Comment:
the valuecheck is in.  yes -- i'm aok with moving it to testUtil.h .  (note that there are a few header inclusions at the top of drakeUtil.h that can move with it, and are commented as such).